kroxylicious-kubernetes-api/pom.xml

```diff
@@ -0,0 +1,231 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I can't comment on a file that doesn't exist, but I think we want a `README.md` for this module which says that we're not on the hook for *Java compatibility* of this API. We care about compatibility of the CRD schemas, and we can't guarantee that the Java generator will generate Java code which is source or binary compatible as those schemas evolve.